examples_utils/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod camera;
```

Comment:
Since this will show up on docs.rs This needs some top level documentation explaining that this package contains code meant to be reused in examples and not necessarily for user code.